tms_objects2.erb

```diff
@@ -223,6 +223,18 @@ xsi:schemaLocation="http://www.lido-schema.org http://www.lido-schema.org/schema
               <lido:conceptID lido:source="AAT" lido:type="local">300054713</lido:conceptID> 
               <lido:term>production</lido:term> 
             </lido:eventType> 
+        <% CreatorSet.each do |c| %> 
+            <lido:eventActor> 
+          <% if c["creatorname"] != '' %> 
+              <lido:displayActorInRole><%= c["creatorname"] %></lido:displayActorInRole> 
+          <% end %> 
```

Comment:
Missing display bio.  like in coboat

{if $DC->DisplayBio != ''}
<lido:displayActorInRole lido:label="New bio line">{$DC->DisplayBio}</lido:displayActorInRole>
{/if}

Reply:
There is an formatting issue the the non- new bio line displayActorInRole, but will tackle that at the end.

Reply:
but logic is wrong, should be:
{if $DC->DisplayBio != ''}
otherwise displays empty element

Reply:
Still displaying when empty, actually should be:
<% unless c["DisplayBio"].nil? %>
nil is different then != ''